app/audit.go

```diff
@@ -14,3 +38,124 @@ func (a *App) GetAudits(userId string, limit int) (model.Audits, *model.AppError
 func (a *App) GetAuditsPage(userId string, page int, perPage int) (model.Audits, *model.AppError) {
 	return a.Srv().Store.Audit().Get(userId, page*perPage, perPage)
 }
+
+func (s *Server) configureAudit(adt *audit.Audit) {
```

Comment:
As discussed the other day, I think the plan is to remove this config sending it to the database, and instead expose some basic config for routing it to external sources.

Reply:
`configureAudit` now uses the config store to fetch options for connecting to remote syslog via TLS.   https://github.com/mattermost/mattermost-server/pull/13967/commits/131b0b37150a629373e2cdabd06bf82a0600dcbe